### PR TITLE
docs(ces): update doc and test of ces alarm rule

### PR DESCRIPTION
**Pull request by Jason-Zhang9309**
**What this PR does / why we need it**:

**Which issue this PR fixes**:
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
fixes #xxx

**Special notes for your reviewer**:

**Release note**:

```release-note

```

## PR Checklist

* [x] Tests added/passed.
* [x] Documentation updated.
* [x] Schema updated.

## Acceptance Steps Performed

```
make testacc TEST='./huaweicloud/services/acceptance/ces' TESTARGS='-run TestAccCESAlarmRule'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud/services/acceptance/ces -v -run TestAccCESAlarmRule -timeout 360m -parallel 4
=== RUN   TestAccCESAlarmRule_basic
=== PAUSE TestAccCESAlarmRule_basic
=== RUN   TestAccCESAlarmRule_withEpsId
=== PAUSE TestAccCESAlarmRule_withEpsId
=== RUN   TestAccCESAlarmRule_sysEvent
=== PAUSE TestAccCESAlarmRule_sysEvent
=== RUN   TestAccCESAlarmRule_allInstance
=== PAUSE TestAccCESAlarmRule_allInstance
=== RUN   TestAccCESAlarmRule_withResourceGroup
=== PAUSE TestAccCESAlarmRule_withResourceGroup
=== RUN   TestAccCESAlarmRule_old
=== PAUSE TestAccCESAlarmRule_old
=== CONT  TestAccCESAlarmRule_basic
=== CONT  TestAccCESAlarmRule_allInstance
=== CONT  TestAccCESAlarmRule_old
=== CONT  TestAccCESAlarmRule_withResourceGroup
--- PASS: TestAccCESAlarmRule_allInstance (15.12s)
=== CONT  TestAccCESAlarmRule_sysEvent
--- PASS: TestAccCESAlarmRule_sysEvent (13.73s)
=== CONT  TestAccCESAlarmRule_withEpsId
--- PASS: TestAccCESAlarmRule_withResourceGroup (192.63s)
--- PASS: TestAccCESAlarmRule_old (206.27s)
--- PASS: TestAccCESAlarmRule_basic (209.11s)
--- PASS: TestAccCESAlarmRule_withEpsId (180.32s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/ces       209.291s
```

### CODE REVIEW

File: docs/resources/ces_alarmrule.md [hunk 1]
```diff
@@ -11,57 +11,98 @@ Manages a Cloud Eye alarm rule resource within HuaweiCloud.
...
+  condition  {
+    period              = 1200
     filter              = "average"
     comparison_operator = ">"
-    value               = 6
+    value               = 6.5
+    unit                = "B/s"
+    count               = 1
+    suppress_duration   = 300
+    metric_name         = "network_outgoing_bytes_rate_inband"
```
Comment (Lance52259) [inline docs/resources/ces_alarmrule.md:50]: Does the `network_outgoing_bytes_rate_inband` is the fixed value? If not, it might be better to use variables?

**Comment by Lance52259** [comment]
The parameter `alarm_level` has been deprecated, but the variable is still exist in the document.

**Comment by Jason-Zhang9309** [comment]
> The parameter `alarm_level` has been deprecated, but the variable is still exist in the document.

the alarm_level in condition should be remained

File: docs/resources/ces_alarmrule.md (hunk 1 above)
Comment (Jason-Zhang9309) [reply docs/resources/ces_alarmrule.md:50]: it's a fixed value

**Pull request merged**